mlflow/models/cli.py

```diff
@@ -155,6 +155,50 @@ def prepare_env(
     ).prepare_env(model_uri=model_uri)
 
 
+@commands.command("generate-dockerfile")
+@cli_args.MODEL_URI_BUILD_DOCKER
+@click.option(
+    "--output-directory",
+    "-d",
+    default="mlflow-dockerfile",
+    help="Output directory where the generated Dockerfile is stored.",
+)
+@cli_args.ENV_MANAGER
+@cli_args.MLFLOW_HOME
+@cli_args.INSTALL_MLFLOW
+@cli_args.ENABLE_MLSERVER
+def generate_dockerfile(
+    model_uri, output_directory, env_manager, mlflow_home, install_mlflow, enable_mlserver
+):
+    """
+    Generates a directory with Dockerfile whose default entrypoint serves an MLflow model at port
+    8080 using the python_function flavor. The generated Dockerfile is written to the specified
+    output directory, along with the model (if specified). This Dockerfile defines an image that
+    is equivalent to the one produced by ``mlflow models build-docker``.
+    """
+    if model_uri:
+        _logger.info("Generating Dockerfile for model %s", model_uri)
+    else:
+        _logger.info("Generating Dockerfile for unspecified model")
```

Comment:
```suggestion
        _logger.info("Generating Dockerfile")
```

Do we need `for unspecified model`?

Reply:
@dbczumar asked "Can we add a small logger.info("Generating Dockerfile) message here? If a model_uri is specified, can we include it in the message?"

Maybe I took too far "if model_uri is specified" part :sweat_smile: 

I think we can just add "Generating dockerfile for model <model_uri>" if it's OK to you?
Than it just will be `None` if model_uri was not specified

Reply:
@anuarkaliyev23 Your latest changes look great! Thanks!